api/files/serializers.py

```diff
@@ -188,8 +188,12 @@ def user_id(self, obj):
 
     def update(self, instance, validated_data):
         assert isinstance(instance, FileNode), 'Instance must be a FileNode'
-        for attr, value in validated_data.items():
-            setattr(instance, attr, value)
+        if len(validated_data) == 1 and 'checkout' in validated_data:
```

Comment:
What happens if you modify two things at once? I don't know if that's currently possible, but if we added another metadata option, my guess is this would break. I think moving the conditional into the for loop is the better way to go.